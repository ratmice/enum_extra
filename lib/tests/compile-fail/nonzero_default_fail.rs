#[allow(unused_imports)]
use enum_extra::NonZeroRepr;
use strum::EnumMetadata;

#[derive(NonZeroRepr, EnumMetadata)]
#[repr(u8)]
enum XYZZZ {
    A,
}


fn main() {
}
