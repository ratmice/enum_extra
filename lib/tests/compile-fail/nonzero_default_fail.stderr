error: proc-macro derive panicked
 --> tests/compile-fail/nonzero_default_fail.rs:5:10
  |
5 | #[derive(NonZeroRepr, EnumMetadata)]
  |          ^^^^^^^^^^^
  |
  = help: message: Enum element defaults to zero
